src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -1393,7 +1393,9 @@ std::shared_ptr<ov::npuw::LLMCompiledModel> ov::npuw::LLMCompiledModel::import_m
             compiled->m_prefill_compiled->m_weights_bank = bank;
 
             compiled->m_kvcache_compiled->finalize_weights_bank();
+            compiled->m_kvcache_compiled->m_import_weights_ctx.reset();
             compiled->m_prefill_compiled->finalize_weights_bank();
+            compiled->m_prefill_compiled->m_import_weights_ctx.reset();
```

Comment:
So why do we need to explicitly reset these? I thought that `WeightContext` don't own the resources anymore, or do they?

Reply:
They keep Constants alive, so I reset just in case. It can be reworked in the future though